src/site/_includes/js/eligibility.js

```diff
@@ -2647,6 +2664,34 @@ function wicResult(input) {
   return program.getResult();
 }
 
+function clipperStartResult(input) {
+  const meetsAgeReq = and(
+    ge(input.age, cnst.clipper.MIN_ELIGIBLE_AGE),
+    le(input.age, cnst.clipper.MAX_ELIGIBLE_AGE));
+
+  const grossLimit = MonthlyIncomeLimits.fromAnnual(
+    cnst.clipper.ANNUAL_INCOME_LIMITS,
+    cnst.clipper.ANNUAL_INCOME_LIMIT_ADDL_PERSON);
+
+  const incomeLimit = grossLimit.getLimit(input.householdSize);
+  const underIncomeLimit = le(grossIncome(input), incomeLimit);
+
+  const noRTC = not(input.existingRtcClipperMe);
+
+  const program = new Program();
+  program.addCondition(new EligCondition(
+    'Be 19-64 years old',
+    meetsAgeReq));
+  program.addCondition(new EligCondition(
+    `Have a household income below ${usdLimit(incomeLimit)} 
+    per month (200% federal poverty level)`,
```

Comment:
To keep things as simple and straightforward as possible for our users, please remove the **(200% federal poverty level)** text.  We will take care of keeping our took up-to-date with the latest federal poverty level, so the user really just needs to know what the dollar amount is.

Reply:
Done in [f3038e7](https://github.com/theunitedeffort/theunitedeffort.org/pull/610/commits/f3038e71984f7e2cbbb53b7d59b5ede13d9315f8)